### ISSUE TITLE
Downloading video from https://vix.com/ has audio and image errors since the ad appears

### ISSUE THREAD

**Issue by itmevn007** [ytdl-org/youtube-dl#32753]
- [x ] I'm reporting a broken site support issue
- [ x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x ] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ x] I've searched the bugtracker for similar bug reports including closed ones
- [x ] I've read bugs section in FAQ

## Verbose log

```
PASTE VERBOSE LOG HERE
```

## Description

Hello I'm new.
I downloaded the video from Vix.com.
If I watch on Vix.com, an ad will appear at the 15th minute. The video I downloaded also had an error from the 15th minute, the sound and image do not match each other, the 15th minute audio is lost, it is pushed. after. Please help me handle this problem. Thank
Here is the video I downloaded: https://vix.com/es-es/video/video-2991097 (need MExico IP vpn).
Here is the M3U8 code: https://dcs-vod.mp.lura.live/vod/p/session/master.m3u8?i=i176980327-n0eb8e8cc-52d0-4094-b199-aba13c3dd5c6&anvtrid=7a92f0d651785cdcf7bb927439a480f7&guid=4 e1b43c2-7c75-90d4 -d6518dd7ea37
I run the following code to download:
youtube-dl -o output9.mp4 "https://dcs-vod.mp.lura.live/vod/p/session/master.m3u8?i=i176980327-n0eb8e8cc-52d0-4094-b199-aba13c3dd5c6&anvtrid=7a92f0d651785cdcf7bb927439a480f7& guid=4e1b43c2 -7c75-90d4-d6518dd7ea37"

**Comment by dirkf** [comment]
Please do **paste the text log of your failing command**, as asked. Maintainers may not be able to reproduce the problem, especially as you say that it needs a Mexico IP. Also, when completing the template, try to put `[x]` and not `[ x]` (etc) for GitHub to recognise the check-mark and display it properly; in future we might add workflow that rejects an issue if GitHub finds unchecked tasks.
 
If you had posted the result of a command to get the Vix page, the log would have included something like this (but with the actual page URL, if you have a Mexico IP):
```console
ERROR: Unsupported URL: https://assets.vix.com/geo.html
```

As for the manifest URL, it may not be downloadable outside the context of the site from which it was captured. It may be time-limited, or require additional HTTP headers, such as `Referer`, `Origin`, `Authorization` or cookies. This is a related [FAQ item](https://github.com/ytdl-org/youtube-dl#user-content-i-extracted-a-video-url-with--g-but-it-does-not-play-on-another-machine--in-my-web-browser).

In summary, duplicate of #29120, where support for vix.com was requested and a poster suggested how to add it (possibly no longer valid). As no-one has written an extractor, the issue remains open.
